wordpress-mysql-replication/azuredeploy.json

```diff
@@ -237,12 +231,12 @@
     },
     "variables": {
         "template": {
-            "base": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/wordpress-mysql-replication"
+            "base": "https://raw.githubusercontent.com/Sunbuild/azure-quickstart-templates/master/wordpress-mysql-replication"
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc for handling artifact uri's